packages/paste-core/components/switch/src/SwitchGroup.tsx

```diff
@@ -0,0 +1,79 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import {InlineControlGroup} from '@twilio-paste/inline-control-group';
+import type {InlineControlGroupProps} from '@twilio-paste/inline-control-group';
+import {SwitchContext} from './SwitchContext';
+
+export interface SwitchGroupProps extends InlineControlGroupProps {
+  name: string;
+  onChange?: (checked: boolean) => void;
```

Comment:
In this case we're typing it differently. Normally onChange events are `event => void`, but in this case we pass `event.target.checked` directly instead of the entire `event`.